pkg/shell/shell.scss

```diff
@@ -353,15 +353,6 @@ thead.credential-lock.credential-unlocked > *:last-child {
     display: table-row !important;
 }
 
-
-/* The server avatar */
-#server-avatar {
```

Comment:
This indeed seems obsolete. We use `#machine-avatar` and various other *-avatar in shell and dashboard, but not this one.